python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -130,6 +131,18 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             }
           ];
 
+    // Construct the custom message from the actor.
+    let actorCustomDisplay: JSX.Element[] = []
+    if (actor.state !== -1 && actor.webuiDisplay) {
+      Object.keys(actor.webuiDisplay).sort().forEach(key =>{
+        actorCustomDisplay.push(<Typography className={classes.webuiDisplay}>
+          &nbsp; &nbsp; {key}: {actor.webuiDisplay![key]}
```

Comment:
The `!` shouldn't be necessary here since we've already checked that `actor.webuiDisplay` is not `undefined`. (Note that you're already using `actor.webuiDisplay` without a `!` in the `Object.keys()` call.)

Reply:
![image](https://user-images.githubusercontent.com/21118851/73110843-b6363100-3ebc-11ea-9ae9-a0c12a6fc1a9.png)
It's funny that somehow the typechecker complained about this case.